.chloggen/forward-beta.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
```

Comment:
Since this is a docs only change, i would remove the changelog here and skip the check

Reply:
Isn't it a user facing concern though?

Reply:
Oh... maybe i'm misunderstanding. Should this change also update `forward.go`? https://github.com/open-telemetry/opentelemetry-collector/blob/3b62976f35387b150244390e8f8286a820966963/connector/forwardconnector/forward.go#L34-L36

Reply:
+1 @codeboten